src/components/TodoFilter.tsx

```diff
@@ -0,0 +1,93 @@
+import { useContext, useMemo } from 'react';
+import { NavLink } from 'react-router-dom';
+import classNames from 'classnames';
+import { DispatchTodo, StateTodo } from '../context';
+import { Status, ActionTypes } from '../types';
+
+type Props = {
+  location: {
+    search: string;
+  }
+};
+
+export const TodoFilter: React.FC<Props> = ({ location }) => {
+  const { todos } = useContext(StateTodo);
+  const dispatch = useContext(DispatchTodo);
+
+  const status = [Status.ALL, Status.ACTIVE, Status.COMPLETED];
+  const naming = ['All', 'Active', 'Completed'];
```

Comment:
It should be moved out of component as its static values
Also my assumtion create 1 arra, tat will contain all required data
```
const FILTER_BUTTONS = [{
  id: Status.ALL,
  caption: 'All'
}, {
  id: Status.ACTIVE,
  caption: 'Active'
}, {
  id: Status.COMPLETED,
  caption: 'Completed'
}]
```

Reply:
🙌🏼 Yeah it's very good idea.Thanks!